english/eating-psychadelics/eating-psychadelics.md

```diff
@@ -0,0 +1,9 @@
+You recall the previous night and how you played some sort of version of beer pong but the cups where filled with a golden syrup sort of liquor.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.